src/devices/Nmea0183/tests/Ais/AddressedSafetyRelatedMessageTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Iot.Device.Nmea0183.Ais;
+using Iot.Device.Nmea0183.AisSentences;
+using Shouldly;
```

Comment:
As you know we usually just use Asserts for our tests. Did you mainly use the dependency for the better messages that you get or is there something in specific that Assert doesn't support? I only ask because it would be good to be somewhat consistent on the repo.

Reply:
I added it because the original repo I took these classes from used it and I didn't want to rewrite all the tests. I didn't know this library before, but I think it's very nice. 

I do agree that this generates a bit of a mess between Assert and ShouldBe-syntax, but is this really an issue? I don't think so, since both variants are very easy to understand (and we should encourage users for more test coverage, anyway...!)